.npmignore

```diff
@@ -1,7 +0,0 @@
-.settings
-.jshintignore
-.jshintrc
```

Comment:
We still want some style hints

Reply:
And this pr isn't removing them... its changing how npm knows what files to publish when we do a release.

Reply:
ok - so still in for a git clone - but not for an npm pack.

Reply:
npm pack obeys `.npmignore` - so these files are already not included today. This PR doesn't change that fact.

Reply:
ok - happy to accept.